pages/01.gantry5/02.configure/01.gantry-admin/docs.md

```diff
@@ -70,13 +70,14 @@ The **Global Admin Panels** do not affect a single Outline, but have influence o
 
 ![Outlines](outlines.jpg?classes=shadow,border)
 
-This panel is where you can manage and create new outlines, each with its own set of layout, style, assignments, and particle settings. There are two types of outlines, **Standard** and **System**.
+An "outline" is a configuration that is assigned to a **Page Type**, to give it unique layout, style, assignments, and particle settings. The Outlines panel is where you can manage and create new outlines. There are two types of outlines: **Standard** and **System**.
```

Comment:
I don't yet know how to assign an outline to a Page Type in Grav. It would be nice to link **Page Type** to that documentation...